app/models/link_audit.rb

```diff
@@ -0,0 +1,50 @@
+class LinkAudit
```

Comment:
What do you think about calling this `LinkFinder` since that's all it's concerned with?

Reply:
I like it! Renamed to `MissingLinkFinder` in 48e0b737b0cf83ed0e61bfab00806e4d896fd9df.

Reply:
Even better!